reference-architectures/deploy-arch-ibm-ocp-quickstart.md

```diff
@@ -0,0 +1,89 @@
+---
+
+copyright:
+  years: 2025
+lastupdated: "2025-09-03"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Prateek Sharma"
+
+# The release that the reference architecture describes
+version: 3.55.6
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: true
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-ocp-vpc-1728a4fd-f561-4cf9-82ef-2b1eeb5da1a8-global
+
+docs: https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/reference-architectures/deploy-arch-ibm-ocp-quickstart.md
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/reference-architectures/deployable-architecture-ocp-cluster-qs.svg
+
+related_links:
+  - title: "Cloud automation for Red Hat OpenShift Container Platform on VPC"
+    url: "https://cloud.ibm.com/docs/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ocp-ra"
+    description: "A deployable architecture solution that is based on the IBM Cloud for Financial Services reference architecture. This solution delivers a scalable and flexible Red Hat OpenShift environment on IBM Cloud Virtual Private Cloud (VPC), with seamless integration to security, observability, and other foundational services for containerized workloads."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Cloud automation for Red Hat OpenShift Container Platform on VPC - QuickStart variation
```

Comment:
Wrong name - Its "Landing zone for containerized applications with OpenShift"

Reply:
Oops, it got missed in qs variation. I have made the changes. Thank you for pointing out.